content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -133,6 +133,39 @@ log audit backend using the following [kube-apiserver][kube-apiserver] flags:
 - `--audit-log-maxbackup` defines the maximum number of audit log files to retain
 - `--audit-log-maxsize` defines the maximum size in megabytes of the audit log file before it gets rotated
 
+{{< note >}}
+In case kube-apiserver is configured as a Pod,remember to mount the hostPath to the localtion of the policy file and log file. For example, 
+`
+--audit-policy-file=/etc/kubernetes/audit-policy.yaml
+--audit-log-path=/var/log/audit.log
+`
+then mount the volumes:
+
+
+```
+- mountPath: /etc/kubernetes/audit-policy.yaml
+      name: audit
+      readOnly: true
+ - mountPath: /var/log/audit.log
+      name: audit-log
+      readOnly: false
+```
+finally the hostPath:
+
+```
+- hostPath:
+      path: /etc/kubernetes/audit-policy.yaml
+      type: File
+    name: audit
+  - hostPath:
+      path: /var/log/audit.log
+      type: FileOrCreate
+    name: audit-log
+    
```

Comment:
please double check the above snippets:

- are the indentation correct?
- do you think the readers know where to put these configs?

Reply:
Yes, the indentation is correct.
yes, since the the config file already has other existing volume mounts

Reply:
Hi @tengqm . This seems to have stalled .. any idea what needs to be done?

Reply:
line 156 and line 160 should have the same indentation, right?

Reply:
Also, please move the code snippet out of the "note" context which is buggy.

Reply:
@tengqm i have removed the note tag and also corrected the indentation

Reply:
The hostPath one still doesn't look like a valid YAML. Please double check. My understanding is that the two examples should be 

```
- mountPath: /etc/kubernetes/audit-policy.yaml
  name: audit
  readOnly: true
- mountPath: /var/log/audit.log
  name: audit-log
  readOnly: false
```

and 

```
- hostPath:
    path: /etc/kubernetes/audit-policy.yaml
    type: File
  name: audit
- hostPath:
    path: /var/log/audit.log
    type: FileOrCreate
  name: audit-log
```